.github/workflows/go.yml

```diff
@@ -43,12 +45,18 @@ jobs:
         make manifests
         git diff --exit-code
 
+    - name: Build image
+      run: |
+        docker build . -t ${built_image}
+
     - name: Create K8s Kind Cluster
-      run: kind create cluster --image kindest/node:v1.19.7
+      run: |
+        kind create cluster --image kindest/node:v1.19.7
+        kind load docker-image ${built_image}
 
     - name: Deploy Metal LB Operator
       run: |
-        make deploy
+        IMG=${built_image} KUSTOMIZE_DEPLOY_DIR="config/kind-ci/" make deploy
```

Comment:
since u set IMG here in the script then we can just undo what @fedepaol did in Makefile and Manager.yaml and it will just pickup the new image can u pls verify

Reply:
can you elaborate? not sure I understand what you are suggesting

Reply:
what I meant is setting IMG env as u have above is all we need to generate operator image and we just need to reset makefile, manager.yaml and the 2 yamls below to what was originally auto generated I am not sure if u addressing this in this PR since u renaming the images or another PR will address this